docs/source/interfaces/indirect/Crystal Field Python Interface.rst

```diff
@@ -466,6 +466,127 @@ can be used like a `CrystalField` object in this way, although `Temperatures` an
 
 
 
+Intensity Scaling
+-----------------
+The `IntensityScaling` parameter allows applying of a scale factor to the data. This scale factor is not applied
+immediately but only when a fit is run. As a consequence, a spectrum plotted after setting an `IntensityScaling`
+parameter but before running the fit will remain unchanged. Unlike other parameters the default value for `IntensityScaling`
+is 1 instead of 0.
+
+There are differences on how to use the `IntensityScaling` parameter(s) depending on the use-case:
+
+Single ions fitted to a single spectrum
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When fitting a single ion symmetry to a single spectrum `IntensityScaling` is set to a single value::
+
+    cf = CrystalField('Ce', 'C2v', B20=0.37737, B22=3.9770, B40=-0.031787, B42=-0.11611, B44=-0.12544, Temperature=44.0,
+    FWHM=1.1, IntensityScaling=3.0)
+
+Instead of initializing `IntensityScaling` in the constructor this parameter can also be set later::
+
+    cf.IntensityScaling = 3.0
+
+In both cases `IntensityScaling` is initially set to a value of 3.0 but can be varied during the fitting process. Only
+with a tie on `IntensityScaling` the value remains fixed::
+
+    cf.ties(IntensityScaling = 3.0)
+
+Single ions fitted to multiple spectra
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In case of multiple spectra the `IntensityScaling` parameter is replaced by a list of values. The size of this list must
+match the size of the `Temperature` list::
+
+    cf = CrystalField('Ce', 'C2v', B20=0.37737, B22=3.9770, B40=-0.031787, B42=-0.11611, B44=-0.12544,
+                  Temperature=[44.0, 50.0], FWHM=[1.1, 0.9], IntensityScaling=[3.0, 0.05])
+
+The parameters for `IntensityScaling` can also be initialized later::
+
+    cf.IntensityScaling=[3.0, 0.05]
+
+When setting a tie the value for an `IntensityScaling` parameter can be set directly using the index of the parameter::
+
+    cf.ties(IntensityScaling1 = 5.0)
+
+In the example above the initial value of the second `IntensityScaling` parameter would be ignored and permanently set
+to 5.
+
+Intensity scaling for multiple ions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+For multiple ions there are two options for creating a `CrystalFieldMultiSite` object. Either two `CrystalField` objects
+are combined or a `CrystalFieldMultiSite` object is created directly.
+Please note that the first `CrystalField` object with a scaling factor is set as ion0 when combining two `CrystalField`
+objects::
+
+    cf = cf1 + 0.5*cf2
+
+creates a `CrystalFieldMultiSite` object with `cf2` as ion0 and `cf1` as ion1. It is worth mentioning that in case of two
+`CrystalField` objects with a scaling factor the scaling factor of the second object is ignored::
+
+    cf = 2*cf1 + 3*cf2
+
+results in the same `CrystalFieldMultiSite` object as::
+
+    cf = 2*cf1 + cf2
+
```

Comment:
Thanks for clarifying this but I think we should mark this as a bug and change the Python code - I thought that the prefactor is used directly as the `abundances` and this is fed through as the ion `IntensityScaling` in C++? I've only glanced through the code, but is the problem with the Python code or the C++?

Taking the example

```
cf = 2*cf1 + 3*cf2
```

The `2*cf1` and `3*cf2` should each result in a `CrystalFieldSite` object. The `+` then calls `__add__` of `CrystalFieldSite` which should respect the abundance (`2` and `3`) of each object - but that's not the case?

For the example

```
cf = cf1 + 0.5*cf2
```

`cf1` is just a plain `CrystalField` object whilst `0.5*cf2` is a `CrystalFieldSite`. Now, I think (from skimming the code) that in this case the `__add__` of `CrystalField` is called which then calls the `__add__` of `CrystalFieldSite` but in doing so swaps it around (so that `cf2` is the left hand side or `ion0`). If instead it calls `__radd__` of `CrystalFieldSite` and we define `__radd__` where `self` is `ion1` instead - maybe that will work?